### ISSUE TITLE
Accessibility: VoiceOver announces unavailable dates

### ISSUE THREAD

**Issue by chica26** [gpbl/react-day-picker#1437]
### Bug description

When navigating through the calendar using VoiceOver the unavailable dates ere announced.

### Expected behavior
VoiceOver should not announce the unavailable dates

Would like to know if this is an issue with the day picker library and if it does not support screen readers to skip the unavailable dates.
